### ISSUE TITLE
Define guidance for cancelling and rescheduling CC touchpoints

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#60851]
## User Story
As a VFS team member, I want to locate the guidance for how to cancel or reschedule my CC touchpoint, so that I can cancel/reschedule correctly.

As a Governance team member, I want to publish guidance for VFS teams to follow on how to cancel or reschedule a CC touchpoint so that all VFS team members can follow the guidance and each cancel/reschedule is handled consistently. 

Assignee: @allison0034 
Peer Reviewer: @briandeconinck 

## Description
When a VFS team schedules an event in Calendly, there is some default language that is added to the event:

> Need to make changes to this event?
Cancel: [https://calendly.com/cancellations/8b797161-f118-468d-a7bd-b9472909aa31](https://www.google.com/url?q=https://calendly.com/cancellations/8b797161-f118-468d-a7bd-b9472909aa31&sa=D&source=calendar&ust=1688217958266205&usg=AOvVaw0fVX15hIfwfXIeu1-zO4FL)
Reschedule: [https://calendly.com/reschedulings/8b797161-f118-468d-a7bd-b9472909aa31](https://www.google.com/url?q=https://calendly.com/reschedulings/8b797161-f118-468d-a7bd-b9472909aa31&sa=D&source=calendar&ust=1688217958266205&usg=AOvVaw06IdMiuY8ysMC79MUOVUD-)

VFS team members have followed the links to cancel or reschedule, but not notified Governance team via other channels. This makes it difficult for Governance to be aware of the change without Shira notifying them since she receives the Calendly emails. Note: This feature can be toggled off in Calendly and we can add a custom input. See image below:
![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/43fecc05-be59-467b-9b10-f0596fcd3de0)

In other scenarios, VFS team  have contacted Governance via Slack or Github to cancel or reschedule a CC touchpoint.

The purpose of this ticket is to define a preferred process for VFS teams to follow when they would like to request a scheduled touchpoint be cancelled or rescheduled for a different date.

## Impacted Artifacts
Calendly 
[New operational artifact](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2737766401/Rescheduling+and+cancelling+touchpoints)

## Tasks
- [x] Determine what process we want VFS teams to follow. Either a) using the Calendly options to reschedule on their own with additional actions to notify Governance or b) notify Governance so Governance can do the rescheduling
   - Feel free to ask other Governance team members their thoughts and opinions
- [x] Once the decision has been made, create documentation drafts to document changes to Calendly and the code blocks. Create a new documentation draft for the operational guidance doc.
- [x] Once peer review is complete, contact Shira or Allison to make changes to Calendly.
- [x] Publish changes to code blocks and publish new operational doc.
- [x] Share new guidance with Governance team members on the #platform-governance-team Slack channel

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] We have defined and implemented a cancellation and reschedule policy for VFS teams to follow
- [x] We have defined and implemented a cancellation and reschedule operational policy for Governance team to uphold

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by briandeconinck** [comment]
On hold for higher-priority #63241

**Comment by allison0034** [comment]
Added "If you cancel or reschedule your meeting, please notify the Governance team by tagging them in your Slack request in your channel." to the Calendly cancellation policy.

**Comment by allison0034** [comment]
Created a [draft for instructions](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2737766401/60851+-+Allison+What+to+do+if+a+review+is+rescheduled+or+cancelled)

**Comment by allison0034** [comment]
This will move into the next sprint as it was just assigned to me on Friday

**Comment by allison0034** [comment]
Reached out to team on [Slack](https://dsva.slack.com/archives/GNF5GSD9N/p1692294941211729) for feedback.

**Comment by shiragoodman** [comment]
@allison0034 FYI i added a [comment to the draft doc](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2737766401/60851+-+Allison+Rescheduling+and+cancelling+touchpoints) (in addition to the comments inline that you already resolved. )

**Comment by shiragoodman** [comment]
@allison0034 FYI i added a [comment to the draft doc](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2737766401/60851+-+Allison+Rescheduling+and+cancelling+touchpoints) (in addition to the comments inline that you already resolved. )

**Comment by allison0034** [comment]
@briandeconinck this is ready for your peer review.

- You will need to schedule and delete a test in the [Midpoint/DI Calendly](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review) to check that a cancellation policy is there.
- You will need to schedule and delete a test in the [Staging Calendly](https://calendly.com/collaboration-cycle/staging-review) to check that a cancellation policy is there.
- Review the new [rescheduling and cancelling touchpoints guidance](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2737766401/Rescheduling+and+cancelling+touchpoints)

**Comment by briandeconinck** [comment]
@allison0034 This all looks great!
